src/pages/office/cat-adi-part2/components/show-me-questions/show-me-questions.ts

```diff
@@ -88,4 +113,29 @@ export class ShowMeQuestionsCatADI2Component implements OnChanges {
   findQuestion(): VehicleChecksQuestion {
     return this.questions.find(question => question.code === this.questionResult.code);
   }
+
+  evaluateLabels(): void {
```

Comment:
name could perhaps be more descriptive, e.g. updateShowMeQuestionAttributes()

Reply:
fine